tests/render.spec.js

```diff
@@ -38,13 +38,17 @@ describe('render', () => {
   })
 
   test('debug', () => {
+    const originalConsole = global.console
+
     global.console = {log: jest.fn()}
 
     const {debug} = render(App)
 
     debug()
 
     expect(global.console.log).toHaveBeenCalledWith(prettyDOM(document.body))
+
+    global.console = originalConsole
```

Comment:
i wasn't able to use `console.log` after this test case, just preserving the env global here

Reply:
Thank you!  I was just thinking about that over breakfast lol